src/core/tooltips/rfc.md

```diff
@@ -0,0 +1,87 @@
+## Tooltip Service
+
+This service allow us show tooltips in different places of our application.
+Underhood it uses `@konturio/ui-kit` tooltip component for show tooltip in theme style
+
+Tooltip:
+
+- shows by hover-in and hides by hover-out events (after delay)
+- Not interactive
+- Can contain only text
+
+## How to use
+
+We assume that all cases of using a tooltip can be reduced to two cases
+
+### We have an element around which a tooltip should be shown
+
+In that case you need import special components and use them
+
+```jsx
+import { Tooltip, TooltipTrigger, TooltipContent } from '~core/tooltip';
+
+function Sidebar() {
+  return (
+    <nav>
+      <Tooltip>
```

Comment:
Looks like TooltipContext or TooltipWrap or smth

Reply:
Not subject of this PR i guess

Reply:
Less words == Simpler

Reply:
Less words === more interpretations 😄

Reply:
I'am not see any way how it can be intepretated here in wrong way